controllers/application_controller.go

```diff
@@ -205,7 +229,20 @@ func (r *ApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 func (r *ApplicationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	gofakeit.New(0)
 
+	mapSecretsFunc := func(obj client.Object) []reconcile.Request {
```

Comment:
So, does this mean HAS would only watch secrets with the name `has-github-token`, and no other secrets would be watched? What if `has-github-token` doesn't exist? Will it still work fine? (e.g. if we're running the HAS binary locally with the `GITHUB_AUTH_TOKEN` env var exported)

Reply:
> So, does this mean HAS would only watch secrets with the name has-github-token, and no other secrets would be watched?

We can extends this function any time to watch more secrets. For now application service uses only one secret to store some part of the configuration.

> What if has-github-token doesn't exist? Will it still work fine? (e.g. if we're running the HAS binary locally with the GITHUB_AUTH_TOKEN env var exported)

Yes, it will work fine with env variable.